pkg/drivers/kic/kic.go

```diff
@@ -109,6 +109,9 @@ func (d *Driver) Create() error {
 	} else if gateway != nil && staticIP != "" {
 		params.Network = networkName
 		params.IP = staticIP
+	} else if gateway == nil && staticIP != "" {
```

Comment:
Instead of adding an additional if statement, the `gateway != nil` from the above if should be removed if we don't care about the state of the gateway.

Reply:
yes, that was the original intention, however, added the condition separately to handle specific use cases that could arise based on the gateway. We could skip and just validate the staticIP check and use it.